pkg/querier/querier.go

```diff
@@ -119,28 +121,31 @@ func New(cfg Config, distributor Distributor, chunkStore ChunkStore) (storage.Qu
 }
 
 // NewQueryable creates a new Queryable for cortex.
-func NewQueryable(dq, cq storage.Queryable, distributor Distributor, cfg Config) storage.Queryable {
+func NewQueryable(distribQuerier, storeQuerier storage.Queryable, chunkIterFn chunkIteratorFunc, cfg Config) storage.Queryable {
```

Comment:
A nit. Technically, I think `distribQuerier` and `storeQuerier` should be called `*Queryable`. Then the querier is instantiated from the queryable.